indexer/kv_indexer.go

```diff
@@ -0,0 +1,230 @@
+package indexer
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
```

Comment:
**nitpick:** This package was deprecated and moved to its own module here "cosmossdk.io/errors"

Reply:
We can migrate that globally sometime.